apidoc/Titanium/App/Properties/Properties.yml

```diff
@@ -157,97 +157,107 @@ methods:
-        
+
   - name: setDouble
     summary: |
-        Sets the value of a property as a double (double-precision, floating point) data type. The 
+        Sets the value of a property as a double (double-precision, floating point) data type. The
         property will be created if it does not exist.
     parameters:
       - name: property
         summary: Name of property.
         type: String
-        
+
       - name: value
         summary: Property value.
         type: Number
-        
+
   - name: setInt
     summary: |
-        Sets the value of a property as an integer data type. The property will be created if it 
+        Sets the value of a property as an integer data type. The property will be created if it
         does not exist.
     description: |
-        Use [getDouble](Titanium.App.Properties.getDouble) and [setDouble](Titanium.App.Properties.setDouble) 
+        Use [getDouble](Titanium.App.Properties.getDouble) and [setDouble](Titanium.App.Properties.setDouble)
         to store values outside the integer data type range of `-2,147,483,648` to `2,147,483,647`.
     parameters:
       - name: property
         summary: Name of property.
         type: String
-        
+
       - name: value
         summary: Property value, within the range `-2,147,483,648` to `2,147,483,647`.
         type: Number
-        
+
   - name: setList
     summary: |
-        Sets the value of a property as an array data type. The property will be created if it 
+        Sets the value of a property as an array data type. The property will be created if it
         does not exist.
     parameters:
       - name: property
         summary: Name of property.
         type: String
-        
+
       - name: value
         summary: Property value.
         type: Array<Object>
-        
+
   - name: setObject
     summary: |
-        Sets the value of a property as an object data type. The property will be created if it 
+        Sets the value of a property as an object data type. The property will be created if it
         does not exist.
     parameters:
       - name: property
         summary: Name of property.
         type: String
-        
+
       - name: value
         summary: Property value.
         type: Object
-        
+
   - name: setString
     summary: |
-        Sets the value of a property as a string data type. The property will be created if it 
+        Sets the value of a property as a string data type. The property will be created if it
         does not exist.
     parameters:
       - name: property
         summary: Name of property.
         type: String
-        
+
       - name: value
         summary: Property value.
         type: String
-        
+
+properties:
+  - name: useEncryption
+    summary: |
+        If enabled it will use Androids [EncryptedSharedPreferences](https://developer.android.com/reference/androidx/security/crypto/EncryptedSharedPreferences)
+        to store properties.
```

Comment:
`EncryptedSharedPreferences` has been deprecated lately, see the link you've used. It's deprecated in 1.1.0-alpha-7 and we should target 1.0.0, but if the APIs in the main SharedPreferences expose the necessary APIs already, those should be used directly, so we don't need to migrate it again.

Otherwise, this PR should wait for a stable 1.1.0.

Reply:
yeah `MasterKey` was added in 1.1.0-alpha01. I'll check for the deprecation message, updated to 1.1.0-alpha07 and I see it now too. We'll have to wait for the final version 1.1.0.